Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1060,6 +1073,24 @@ - (BNCLinkData *)prepareLinkDataFor:(NSArray *)tags andAlias:(NSString *)alias a
 }
 
 
+#pragma mark - BranchUniversalObject methods
+
++ (Branch *)getCurrentInstanceIfAny {
+    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper preferenceHelper];
+    return [Branch getInstanceInternal:preferenceHelper.lastRunBranchKey returnNilIfNoCurrentInstance:NO];
+}
+
```

Comment:
I think you mean yes as the argument here but I disagree that we shouldn't do anything if the simpleton isn't initialized. More on this below.